flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/Execution.java

```diff
@@ -219,7 +212,7 @@ public Execution(
 		this.stateTimestamps = new long[ExecutionState.values().length];
 		markTimestamp(CREATED, startTimestamp);
 
-		this.partialInputChannelDeploymentDescriptors = new ConcurrentLinkedQueue<>();
+		this.partitionInfos = new ArrayList<>();
```

Comment:
It is good practice to specify an initial capacity for collections.

Reply:
How about initializing 10 here?